website/index.jade

```diff
@@ -12,57 +12,115 @@ html
     section.hero
       .container
         .left
-          img(src="img/git-town-horizontal-w.svg")
-          h2 What is it?
-          p Gittown is an open-source Git plugin that makes using Git on the command line even more productive and convenient! It provides additional high-level Git commands that perform typical steps in collaborative software development workflows.
+          h2
+            | High-level <br>
+            | command line interface
+            | for Git
+          p
+            | Git Town is an open-source Git plugin.<br>
+            | It adds Git commands
+            | that make collaborative software development
+            | more efficient and safe.
 
-        iframe.intro-video(src="https://drive.google.com/a/originate.com/file/d/0BxFxxboeZOPlcjRiVjV5b0c4Qmc/preview" width="550" height="300" frameborder="0")
+          .buttons
+            a.button.learn-more(href="/install.html") Install
+
+
+        iframe.intro-video(src="https://drive.google.com/a/originate.com/file/d/0BxFxxboeZOPlWjFkM1BPek9MUFE/preview" width="550" height="300" frameborder="0")
 
     section.tagline
-      h2 A better way to Git
-      p Git Town adds high-level workflow commands to Git. These commands make it super easy to create, merge, synchronize, and clean up feature branches. They are robust, safe, and well-tested.
+      h2 A workflow layer on top of vanilla Git
+      p
+        | Git is a great <i>foundation</i> for source code management. <br>
+        | It provides all required pieces of functionality, <br>
+        | and supports many different ways of using it equally well. <br>
+
+      p
+        | It is intentionally generic and basic, though. <br>
+        | Using it properly as a developer is cumbersome and repetitive. <br>
+        | For example, correctly merging a finished feature branch
+        | requires <br>
+        | up to <a href="https://github.com/Originate/git-town/blob/master/features/git-ship/supplied_branch/feature_branch/with_tracking_branch.feature#L19-L33">
+        | <b>15</b></a> individual Git commands!
+
+      p
+        | Git Town adds a layer of high-level commands to Git. <br>
+        | They perform typical development workflow operations <br>
+        | like creating or shipping feature branches, <br>
+        | similar to how an expert Git user would do it.
+
+      p All while being as generic and unopinionated as Git.
+
+      .buttons
+        a.button.view-more(href="https://drive.google.com/a/originate.com/file/d/0BxFxxboeZOPlWjFkM1BPek9MUFE/view" target="_blank") View screencast
+        | &nbsp; &nbsp; &nbsp; &nbsp;
```

Comment:
You will need to add a class underneath `section.tagline` (line 72) in `_home.styl`. Remove the `&nbsps;` (You're making Alex and I cry) and add something like:

```
section.tagline
  a.learn-more
    margin-left: 20px
```